tests/test_read_write/test_write_jsonl.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+import os
+from clumper import Clumper
+
+
+def test_local_write_exists(tmp_path):
+    """Test that an error is raised if the written file JSONL doesn't exists."""
+    path = str(tmp_path / "cards_copy.jsonl")
+    clump = Clumper.read_jsonl("tests/data/cards.jsonl")
+    clump.write_jsonl(path)
+    assert os.path.exists(path)
```

Comment:
Technically this test is redundant because the test below also captures it. But it's not a reason for me not to hit merge.